README.md

```diff
@@ -1,2 +1,30 @@
-# workshop
-This is a workshop to teach you modules packaging.
+# Modularity workshop
+
+## High-level agenda
+
+1. 1-minute-intro to modularity
+
+2. `modulemd.yaml`
+ * Guidelines
+ * Best practices (do we have this? is this guidelines?)
+ * Buildtime dependencies
+   * List of well-known modules (What is the status of platform module? I don't want to teach base-runtime)
+   * What components are part of a specific module? (What is the status of this tooling?)
```

Comment:
As I said, he's working on it during this Sprint, Trello #1428 . https://github.com/modularity-modules/python2 shows an example of what he's planning to do.